.github/workflows/ci.yml

```diff
@@ -207,8 +225,8 @@ jobs:
           echo "::warning::Compiler: $(${{ matrix.cxx_compiler }} --version)"
 
           if [[ "${{ matrix.backend }}" == "dpcpp" ]]; then
-            make_targets="build-onedpl-sycl_iterator-tests"
-            ctest_flags="-R sycl_iterator_.*\.pass"
+            make_targets="build-onedpl-sycl_iterator-tests std_ranges.pass"
```

Comment:
I think it would be an additional job which tests only std_ranges.pass and enables C++20.